shadow-cljs.edn

```diff
@@ -62,6 +62,8 @@
          :output-to       "static/tests.js"
          :closure-defines {frontend.util/NODETEST true}
          :devtools        {:enabled false}
+         ;; disable :static-fns to allow for with-redefs and repl development
+         :compiler-options {:static-fns false}
```

Comment:
I had to disable static-fns as shadow has a different default than the cljs compiler for dev. More about this option at https://clojurescript.org/reference/compiler-options#static-fns